src/renderer/assets/styles/index.css

```diff
@@ -32,24 +33,61 @@
   --codeBgColor: #d8d8d869;
   --codeBlockBgColor: rgba(0, 0, 0, 0.03);
   --inputBgColor: rgba(0, 0, 0, .06);
+  --focusColor: var(--themeColor);
```

Comment:
Can we add some docs about the CSS variables? maybe in other PR.

Reply:
Yes, we should add docs for this but I think you'll change a few thing for theme support?